--- ./compat_dcache.h	2014-07-28 14:38:17.432007648 +0000
+++ ./compat_dcache.h	2014-07-28 14:40:46.287013940 +0000
@@ -51,7 +51,7 @@
 /*
  * d_count field was removed in 3.11.0.
  */
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 11, 0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 10, 0)
 #define compat_d_count(dentry) d_count(dentry)
 #elif LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 38)
 #define compat_d_count(dentry) dentry->d_count
